plugins/rest/komand_rest/actions/get/action.py

```diff
@@ -14,13 +14,16 @@ def __init__(self):
 
     def run(self, params={}):
         headers = params.get(Input.HEADERS, {})
-        body = params.get(Input.BODY)
+        body_non_array = params.get(Input.BODY)
+        body_as_an_array = params.get(Input.BODY_AS_AN_ARRAY)
 
-        if body and check_headers_for_urlencoded(headers):
-            body = convert_body_for_urlencoded(headers, body)
+        data = determine_body_type(body_non_array, body_as_an_array)
+
+        if type(data) is dict and check_headers_for_urlencoded(headers):
```

Comment:
I think that would be better to use the `isinstance` method 😄 
```suggestion
        if isinstance(data, dict) and check_headers_for_urlencoded(headers):
```